crates/iota/tests/cli_tests.rs

```diff
@@ -5848,10 +5848,7 @@ async fn test_ptb_sender() -> Result<(), anyhow::Error> {
 async fn test_tree_shaking_package_with_unpublished_deps() -> Result<(), anyhow::Error> {
     let mut test = TreeShakingTest::new().await.unwrap();
     // A package and with unpublished deps
-    let (package_id, _) = test
-        .publish_package("H_depends_on_G_unpublished", true)
-        .await
-        .unwrap();
+    let (package_id, _) = test.publish_package("H", true).await.unwrap();
```

Comment:
Not sure where this was added, but the upstream PR also has a test with `test.publish_package("H", false)`

Reply:
Yeah I noticed, will fix it